src/index.js

```diff
@@ -25,27 +25,61 @@ function setUrl(url, type='push') {
 }
 
 
+function getCurrentLocation() {
+	return (customHistory && customHistory.location) ||
```

Comment:
Implementation is good, but if it's only ever called from `getCurrentUrl()`, let's just inline it there.